src/screens/AccountNetworkChooser.js

```diff
@@ -16,118 +16,241 @@
 
 'use strict';
 
-import React from 'react';
-import { ScrollView, StyleSheet, Text } from 'react-native';
-import { Subscribe } from 'unstated';
+import React, { useState } from 'react';
+import { ScrollView, StyleSheet, Text, View } from 'react-native';
+import { withNavigation } from 'react-navigation';
+
 import colors from '../colors';
-import fonts from '../fonts';
-import TouchableItem from '../components/TouchableItem';
+import AccountCard from '../components/AccountCard';
+import Button from '../components/Button';
 import {
 	NETWORK_LIST,
 	UnknownNetworkKeys,
-	SubstrateNetworkKeys
+	SubstrateNetworkKeys,
+	NetworkProtocols
 } from '../constants';
-import AccountsStore from '../stores/AccountsStore';
-import { empty } from '../util/account';
+import { navigateToPathsList, unlockSeed } from '../util/navigationHelpers';
+import { withAccountStore } from '../util/HOC';
+import { alertPathDerivationError } from '../util/alertUtils';
+import {
+	getAvailableNetworkKeys,
+	getPathsWithSubstrateNetwork
+} from '../util/identitiesUtils';
+import testIDs from '../../e2e/testIDs';
+import ButtonMainAction from '../components/ButtonMainAction';
+import ScreenHeading from '../components/ScreenHeading';
+import Separator from '../components/Separator';
+import fontStyles from '../fontStyles';
 
-export default class AccountNetworkChooser extends React.PureComponent {
-	static navigationOptions = {
-		headerBackTitle: 'Back',
-		title: 'Choose a network'
-	};
-	render() {
+function AccountNetworkChooser({ navigation, accounts }) {
```

Comment:
on start, with clean build, getting a undefined error:

<img width="327" alt="Screenshot 2019-11-18 at 11 03 16" src="https://user-images.githubusercontent.com/10432070/69043456-5a01ed00-09f3-11ea-85bf-5c9485a24dbe.png">

Reply:
Do you have installed the app in this branch before?